supreme/src/commonMain/kotlin/at/asitplus/signum/supreme/mac/MAC.kt

```diff
@@ -0,0 +1,34 @@
+package at.asitplus.signum.supreme.mac
+
+import at.asitplus.KmmResult
+import at.asitplus.catching
+import at.asitplus.signum.indispensable.mac.HMAC
+import at.asitplus.signum.indispensable.mac.MAC
+import at.asitplus.signum.indispensable.misc.xor
+import at.asitplus.signum.supreme.hash.digest
+
+
+fun MAC.mac(key: ByteArray, msg: ByteArray) = mac(key, sequenceOf(msg))
+fun MAC.mac(key: ByteArray, msg: Iterable<ByteArray>) = mac(key, msg.asSequence())
+
+private val HMAC.B get() = digest.inputBlockSize.bytes.toInt()
```

Comment:
What is `B` here? It looks like a type, but I'll guess it is not

Reply:
block length. renamed it